support-frontend/assets/pages/digital-subscription-landing/components/paymentSelection/paymentSelection.jsx

```diff
@@ -10,23 +10,26 @@ import ProductOption, {
   ProductOptionOffer,
   ProductOptionButton,
 } from 'components/productOption/productOption';
+import { mapStateToProps } from './helpers/paymentSelection';
+import { type PaymentOption } from './helpers/paymentSelection';
+import { type Option } from 'helpers/types/option';
 
 // styles
 import './paymentSelection.scss';
 
-import { mapStateToProps } from './helpers/paymentSelection';
-import { type PaymentOption } from './helpers/paymentSelection';
-
 type PropTypes = {
   paymentOptions: Array<PaymentOption>,
+  pageType: Option<string>,
```

Comment:
As mentioned elsewhere I think we should probably use `dailyEditionsVariant` for this now